localization/pose_estimator/ndt_scan_matcher/ndt/CMakeLists.txt

```diff
@@ -1,21 +1,17 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.5)
 project(ndt)
 
-add_compile_options(-std=c++14)
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wno-unused-parameter -Wall -Wextra -Wpedantic -Werror)
+endif()
 
-find_package(catkin REQUIRED COMPONENTS
-  ndt_pcl_modified
-  ndt_omp
-)
-find_package(PCL REQUIRED COMPONENTS common)
-
-catkin_package(
-  INCLUDE_DIRS include
-)
-
-include_directories(include ${catkin_INCLUDE_DIRS} ${PCL_INCLUDE_DIRS})
-link_directories(${PCL_LIBRARY_DIRS})
-add_definitions(${PCL_DEFINITIONS})
+find_package(ament_cmake REQUIRED)
+find_package(PCL REQUIRED COMPONENTS common io registration)
```

Comment:
Interestingly, I got an error when PCL came after ndt_omp and ndt_pcl_modified:
```

CMake Error at CMakeLists.txt:16 (add_library):
  Target "ndt" links to target "ndt_omp::align" but the target was not found.
  Perhaps a find_package() call is missing for an IMPORTED target, or an
  ALIAS target is missing?


CMake Generate step failed.  Build files cannot be regenerated correctly.
```
Not sure why.